java-checks/src/main/java/org/sonar/java/checks/unused/UnusedPrivateMethodCheck.java

```diff
@@ -43,7 +45,9 @@
 import org.sonar.plugins.java.api.tree.NewClassTree;
 import org.sonar.plugins.java.api.tree.ParameterizedTypeTree;
 import org.sonar.plugins.java.api.tree.Tree;
+import org.sonar.plugins.java.api.tree.Tree.Kind;
```

Comment:
This is an unused import (see SQ analysis). Note that you can configure eclipse to remove them.